workflow/management/commands/alias_country_admin_emails.py

```diff
@@ -0,0 +1,68 @@
+from django.core.management.base import BaseCommand
+from workflow.models import CountryAccess
+import socket
+
+
+class Command(BaseCommand):
+    help = "Command to alias country admin emails with the tola@mercycorps.org email"
+    production_host_name = 'neikeata'
+    alias_email = 'tola+'
+    domain = '@mercycorps.org'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--execute', action='store_true', help='Without this flag, the command will only be a dry run'
+        )
+        parser.add_argument(
+            '--supress_output', action='store_true', help='Hide text output'
+        )
+
+    def get_country_admins(self):
+        return CountryAccess.objects.filter(role='basic_admin').exclude(tolauser__user__is_superuser=True)
+
+    def slugify_non_mc_email(self, email):
+        chars = ['@', '.']
+        for char in chars:
+            email = email.replace(char, '_')
+        return email
+
+    def handle_non_mc_email(self, email):
+        """
+        Slugify non mercycorps emails then add the @mercycorps.org domain
+        """
+        email = self.slugify_non_mc_email(email)
+        email += self.domain
+        return email
+
+    def handle(self, *args, **options):
+        """
+        Loops through all country admins and prefixes each email with tola+
+        """
+        emails_aliased = 0
+
+        if not options['supress_output'] and not options['execute']:
+            print('Missing execute flag. This will be a dry run.')
```

Comment:
Without looking at the source I don’t know what happens during a “dry run” (nothing, apparently?). One thing we could is pipe the output (or first `n` lines) to STDOUT